Header header

float64 percent
uint16 vehicle_id
